controller/api/destination/endpoint_profile_translator_test.go

```diff
@@ -1,8 +1,10 @@
 package destination
 
 import (
+	"errors"
```

Comment:
Hmm, for some reason this got picked up even though it's not part of this change. Do you mind pulling latest main again?

```
git pull <upstream> main
```

should do the trick. If it doesn't, you can perhaps do a soft reset and checkout the file.